src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -54,6 +54,12 @@
 #include <time.h>
 #endif
 
+#ifdef TARGET_APPLE
+#include <sys/proc_info.h>
+#include <libproc.h>
```

Comment:
unfortunately libproc isn't allowed on the AppStore ... https://github.com/dotnet/runtime/pull/63934

Reply:
We just need to get path of the current executable. There are multiple ways to do that (eg. _NSGetExecutablePath, getting it from bundle property, etc.). I'll try to rewrite it.

Reply:
_NSGetExecutablePath should work since we're using it in https://github.com/dotnet/runtime/blob/main/src/native/minipal/getexepath.h. Using that helper might actually be preferable anyway.